### PR TITLE
implemented add employee feature

### LINKED ISSUE #105: add employee feature

### PR DESCRIPTION
**Pull request by Parasite-231**
- closes #105

### CODE REVIEW

File: server/controller/admin/user/userController.js [hunk 1]
```diff
@@ -38,4 +36,98 @@ const getProfileDataOfAllExistingUser = async (req, res) => {
...
-module.exports = { getProfileDataOfAllExistingUser };
+const addEmployee = async (req, res) => {
+  const SECRET_KEY = process.env.SECRET_KEY;
+
+  try {
+    const {
+      name,
+      phone,
+      email,
+      personal_email,
```
Comment (asif-bhuti) [inline server/controller/admin/user/userController.js:47]:
```suggestion
      workEmail,
      personalEmail,
```
Name it so it is distinguishable. And please follow the 'camel casing' convention for compound words.

also, **you probably forgot to put the address field**

File: server/model/userModel.js [hunk 2]
```diff
@@ -37,6 +37,16 @@ const userSchema = new Schema({
       "Please enter a valid email",
     ],
   },
+  personal_email: {
```
Comment (asif-bhuti) [inline server/model/userModel.js:40]: use 'camel casing' here as well

Comment (asif-bhuti) [reply server/model/userModel.js:40]: as well as add **address field**

**Review by asif-bhuti — CHANGES_REQUESTED** [review]
Requested the changes in the comments

File: server/controller/admin/user/userController.js (hunk 1 above)
Comment (Parasite-231) [reply server/controller/admin/user/userController.js]: updated

**Review by asif-bhuti — APPROVED** [review]

**Pull request merged**